### PR TITLE
Removal of build warnings

### PR DESCRIPTION
**Pull request by pahor167**
### Description

Removal of build warnings as much as possible. Since we are using both Solidity 0.5 and 0.8 it is unfortunately not possible to remove all warnings since one breaks other.

### CODE REVIEW

File: packages/protocol/contracts/common/FeeHandler.sol [hunk 1]
```diff
@@ -681,7 +682,7 @@ contract FeeHandler is
       "Can't distribute to the zero address"
     );
     IERC20 token = IERC20(tokenAddress);
-    uint256 tokenBalance = token.balanceOf(address(this));
+    token.balanceOf(address(this));
```
Comment (m-chrzan) [inline packages/protocol/contracts/common/FeeHandler.sol]: Can this line be removed?

File: packages/protocol/test-sol/integration/CompileValidatorMock.t.sol [hunk 2]
```diff
@@ -8,7 +8,7 @@ import "forge-std/console.sol";
 import "@test-sol/unit/governance/validators/mocks/ValidatorsMock.sol";
 
 contract CompileValidatorMock is Test {
```
Comment (m-chrzan) [inline packages/protocol/test-sol/integration/CompileValidatorMock.t.sol:10]: Is this test suite useful for anything, could it be completely removed?

**Review by m-chrzan — APPROVED** [review]
Looks good, just two additional suggestions.

File: packages/protocol/contracts/common/FeeHandler.sol (hunk 1 above)
Comment (pahor167) [reply packages/protocol/contracts/common/FeeHandler.sol:685]: This is necessary to forge forge to compile import "@test-sol/unit/governance/validators/mocks/ValidatorsMock.sol";

File: packages/protocol/test-sol/integration/CompileValidatorMock.t.sol (hunk 2 above)
Comment (pahor167) [reply packages/protocol/test-sol/integration/CompileValidatorMock.t.sol:10]: This is necessary for forge to compile
import "@test-sol/unit/governance/validators/mocks/ValidatorsMock.sol";

File: packages/protocol/contracts/common/FeeHandler.sol (hunk 1 above)
Comment (pahor167) [reply packages/protocol/contracts/common/FeeHandler.sol]: I believe the original idea was to test if tokenAddress implements IERC20

Comment (pahor167) [reply packages/protocol/contracts/common/FeeHandler.sol]: On second though - removed

**Pull request merged**